core/CMakeLists.txt

```diff
@@ -42,7 +43,59 @@ if (ENABLE_ASAN OR ENABLE_UBSAN)
   string(REPLACE ";" "," FSANITIZE_STR "${FSANITIZE}")
   add_compile_options(-fsanitize=${FSANITIZE_STR} ${EXTRA_SANITIZER_FLAGS})
   add_link_options(-fsanitize=${FSANITIZE_STR})
-endif()
+endif ()
+
+if (ENABLE_COVERAGE)
+  message(STATUS "Enabling code coverage")
+  if (CMAKE_C_COMPILER_ID MATCHES "Clang") # "Clang" or "AppleClang"
+    # To generate a coverage report from running './subzero --checks-only' using clang:
```

Comment:
Can this go in the README or docs?

Reply:
Sure. However, our README.md is very thin and basically refers to subzero.readthedocs.io. Should I put the coverage stuff in there?

Reply:
Yes good point. I think it would be great to add it as a section in the readthedocs. Wdyt?

Reply:
Sure. I'll do it as a follow-up. And/or move the documentation to github as we discussed.